docs/api/icon.md

```diff
@@ -12,7 +12,7 @@ title: 'ion-icon'
 
 Icon is a simple component made available through the <a href="https://ionic.io/ionicons">Ionicons</a> library, which comes pre-packaged by default with all Ionic Framework applications. It can be used to display any icon from the Ionicons set, or a custom SVG. It also has support for styling such as size and color.
 
```

Comment:
~ Should we just direct link them to the appropriate section? e.g.: https://ionic.io/ionicons/usage#basic-usage

Right now if you click the link you have to know to click "Usage" and find the correct section.

Reply:
Tweaked in https://github.com/ionic-team/ionic-docs/pull/2816/commits/69fb914bc9ab1b049d81ab70de4c47eb761d1a0e.